solidity/hardhat.config.ts

```diff
@@ -54,7 +54,7 @@ const config: HardhatUserConfig = {
       initialBaseFeePerGas: 1000000000,
       forking:
         process.env.FORKING === "true"
-          ? { url: MAINNET_RPC_URL, blockNumber: 20614139 }
+          ? { url: MAINNET_RPC_URL, blockNumber: 20937910 }
```

Comment:
What about adding a code comment about what happened at that particular block?

Reply:
f069bf7902d615c531a53e22580d5324fdeb4861